src/webpki.rs

```diff
@@ -305,5 +305,9 @@ pub struct TrustAnchor<'a> {
 
     /// The value of a DER-encoded NameConstraints, containing name
     /// constraints to apply to the trust anchor, if any.
-    pub name_constraints: Option<&'a [u8]>
+    pub name_constraints: Option<&'a [u8]>,
+
+    /// The root's expiry in seconds since the UNIX epoch. The anchor
+    /// is only considered when the validation time is before this value.
```

Comment:
This is a good optimization and I appreciate you taking the time to do it.

The `Timespec` documentation refers to "the epoch" but AFAICT it isn't necessarily the UNIX epoch, so saying "UNIX epoch" in the comment seems wrong.

We could take the optimization further by making `not_after` a `u32` by reducing its resolution from seconds to days, right? It probably would only matter for 32-bit platforms. Maybe there's not an easy way to do get go from a `TimeSpec` to days since an epoch, thought.